src/WaterPokemon.java

```diff
@@ -0,0 +1,19 @@
+public class WaterPokemon extends Pokemon{
+    private final String waterAttack;
+
+    public WaterPokemon(String name, int level, int hp, int attackStrength, String waterAttack) {
+        super(name, level, hp, attackStrength);
+        super.setType("Water");
+
+        this.waterAttack = waterAttack;
+    }
+
+    @Override
+    public void chooseYou() {
+        System.out.println(super.getName() + " I choose you!");
+    }
```

Comment:
Je schrijft in alle sub classes precies dezelfde chooseYou() je kan deze dan net zo goed definiëren in je super class